### DISCUSSION TITLE
Get data from block_editor in SEO trait

### DISCUSSION THREAD

**Discussion by NorraPL** [area17/twill#1075] [Q&A] [answered]
Hi, 
I have a small question - how can i get a data from custom block editor input (for exampe: local_seo_titile) in Seo Trait/Controller?
@sauron @ifox @yanhao-li

**Comment by pboivin** [comment, marked as answer]
Hey @EPInformatyka,

If I understand correctly, you want to get some data that exists in a block and use it from a controller, not from a blade view. This will depend on the structure of your project, but here's a quick example that should get you going.

Let's say you have an instance of a `Project` model and want to fetch the `title` property from the first block:

```
if ($block = $project->blocks()->first()) {
    $title = $block->content['title'] ?? $defaultTitle;
}
```
